toolkits/nature/packages/nature-article/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@springernature/nature-article",
-  "version": "0.4.0",
+  "version": "0.4.1",
```

Comment:
I'd say this is a major version because we might be including a lot of new stuff right?

Reply:
yeah I guess, will up date. thanks.

Reply:
You don't really need to major version if it's still on version 0 (development).
If this is used in production then it should _already_ be a major version.

Reply:
Have pushed it up to 0.5.0, is that OK ?

Reply:
I should really have done this as part of the previous PR